### ISSUE TITLE
Several features should be converted to Handlebars to align with the workspace

### ISSUE THREAD

**Issue by christiemolloy** [patternfly/patternfly-next#713]
Alert Component and Alignment/Accessibility Utility should be written in handlebars to align with the rest of the workspace, currently it is written in HTML.
